<?xml version="1.0"?>
<robot name="knife_metal"> 
  <link name="base_link">
  <contact>
      <lateral_friction value="1"/>
  </contact>
    <inertial>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <mass value="0.0478" /> 
      <inertia ixx="0" ixy="0" ixz="0" iyy="0" iyz="0" izz="0" />
    </inertial>
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
          <mesh filename="knife_metal.obj"/>
      </geometry>
      <material name="blue">
        <color rgba="0.2, 0.6, 1.0, 1" />
      </material>
    </visual>
    <collision>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
          <mesh filename="knife_metal.obj"/>
      </geometry>
    </collision>
  </link>

  <link name="tip"/>
  <joint name="tip_joint" type="fixed">
    <origin rpy="0 0 0" xyz="-0.14 -0.03 0"/>
    <axis xyz="0 0 1"/>
    <parent link="base_link"/>
    <child link="tip"/>
  </joint>

  <link name="com"/>
  <joint name="com_joint" type="fixed">
    <origin rpy="0 0 0" xyz="-0.05 0 0"/> <!-- NEED TO DOUBLE CHECK-->
    <axis xyz="0 0 1"/>
    <parent link="base_link"/>
    <child link="com"/>
  </joint>


</robot>
